include/host/wasi/environ.h

```diff
@@ -1265,6 +1279,20 @@ Environ::acquirePoller(Span<__wasi_event_t> Events) noexcept {
   return Poller;
 }
 
+inline WasiExpect<EVPoller> Environ::acquirePollerOnly() noexcept {
```

Comment:
This function only provides one feature: take a poller from the existing poller pool or create a new one. However, after calling this function, it will try to close an FD without other features.

Instead of providing this function, I think a new function called `close` is cleaner. The `close` function will iterate all pollers in the poller pool and call `close` to close the corresponding FD.